README.md

```diff
@@ -1,13 +1,12 @@
 <p align="center">
-  <img src="https://docs.kolena.io/assets/images/wordmark-violet.svg" width="400" alt="Kolena" />
+  <img src="https://docs.kolena.io/assets/images/kolena-banner.png" width="640" alt="Kolena" />
```

Comment:
Note that this won't show until the docs are deployed. Planning to manually deploy after landing. Here's the expected outcome:

![Screen Shot 2023-07-07 at 4 26 15 PM](https://github.com/kolenaIO/kolena/assets/9648886/614f4e46-6ec3-46b1-8c0b-a8f2c5620192)